README.md

```diff
@@ -17,3 +17,11 @@ You'll need
 [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm). Run
 `npm install` once. Use `npm run dev` to work locally. To auto-format code,
 `npm run fmt`.
+
+## Tests
+
+Run all tests: `npm run test`
+
+To develop a test interactively, `npm run test -- --debug`. You can step
```

Comment:
![Screenshot from 2023-03-18 18-41-59](https://user-images.githubusercontent.com/1664407/226129706-a6300592-2c62-4334-be52-a18bc2eb0347.png)
This thing is seriously cool